test/alternator/test_ttl.py

```diff
@@ -402,6 +402,38 @@ def test_ttl_expiration_range(dynamodb):
         assert not 'Item' in table.get_item(Key={'p': p, 'c': c1})
         assert 'Item' in table.get_item(Key={'p': p, 'c': c2})
 
+# In the above key-attribute tests, the key attribute we used for the
+# expiration-time attribute had the expected numeric type. If the key
+# attribute has a non-numeric type (e.g., string), it can never contain
+# an expiration time, so nothing will ever expire - but although DynamoDB
+# knows this, it doesn't refuse this setting anyway - although it could.
+# This test demonstrates that:
+@pytest.mark.veryslow
+@pytest.mark.xfail(reason="TTL not implemented yet #5060")
+def test_ttl_expiration_hash_wrong_type(dynamodb):
+    if is_aws(dynamodb):
+        duration = 900
+    else:
+        duration = 3
```

Comment:
ditto №3 - also, is there any particular reason why it's 900 here and 1200 in other test cases?

Reply:
The extreme slowness of the TTL taking effect on DynamoDB seems to depend on the schema of the table, so some tests were even slower than others. In one of the tests even 20 minutes consistently wasn't enough, so I changed it into an hour.  In any case, what really matters is the !is_aws case - nobody will run (or, will very rarely run) these tests on AWS again after the initial development.

By the way, note also that some tests have max_duration - there we can even list 10 hours and the test will finish when it finishes, probably faster - and some tests have "duration" where the test will last exactly that given time, so we want to make that time as low as possible.

Another by the way, running all the tests on Scylla isn't that "very slow" any more - it currently takes 18 seconds for all the ttl tests to finish, and I can possibly make it even faster. At some point I will need to consider to enable them by default, and not mark them "very slow" when running on Scylla.